app/dashboard/management/commands/hackaton_daily_emails.py

```diff
@@ -0,0 +1,48 @@
+'''
+    Copyright (C) 2019 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+import datetime
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from dashboard.models import Activity, Bounty, BountyFulfillment, Profile
+from marketing.mails import bounty_feedback, bounty_not_submitted
+
+
+class Command(BaseCommand):
+    help = 'sends more timed hackathon emails'
```

Comment:
description needs to be more descriptive !

Reply:
Done.